features/data_converter/empty/feature.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from google.protobuf.json_format import Parse, MessageToJson
+from temporalio import workflow
+from temporalio.api.common.v1 import Payload
+from temporalio.api.enums.v1 import EventType
+from temporalio.client import WorkflowHandle
+
+from harness.python.feature import Runner, register_feature
+
+
+@workflow.defn
+class Workflow:
+    """
+    run a workflow that returns nothing
+    """
+
+    @workflow.run
+    async def run(self) -> None:
+        return
+
+
+async def check_result(runner: Runner, handle: WorkflowHandle) -> None:
+    # verify client result is None
+    result = await handle.result()
+    assert result == None
+
+    # get result payload of WorkflowExecutionCompleted event from workflow history
+    history = await runner.get_history_events(handle)
+    e = next(
+        e
+        for e in history
+        if e.event_type == EventType.EVENT_TYPE_WORKFLOW_EXECUTION_COMPLETED
+    )
+    payload = e.workflow_execution_completed_event_attributes.result.payloads[0]
+
+    # load JSON payload from `./payload.json` and compare it to JSON representation of result payload
+    with open(
+        os.path.join(os.path.dirname(runner.feature.file), "payload.json"),
```

Comment:
Is it really worth an entire file just to check empty data and a specific metadata value?

Reply:
I think so

Reply:
If we had cross SDK testing capabilities, I'd say we don't have to test the generated payloads but it's nice to know that null represented the same in all SDKs.